.github/workflows/sanity.yaml

```diff
@@ -28,5 +28,8 @@ jobs:
         with:
           go-version-file: "go.mod"
 
+      - name: Install dependencies required for custom-lint
+        run: sudo apt-get update && sudo apt-get install -y libgpgme-dev
+
```

Comment:
This make me think we need to set `-tags=$(GO_BUILD_TAGS)` when your custom linter is called. We shouldn't need this library.

Reply:
That makes 100% sense.
And seems that worked. 
thank you !!!